trunk/registry/src/routes.rs

```diff
@@ -19,10 +18,9 @@ pub async fn get_all_extensions(
     // Create a transaction on the database, if there are no errors,
     // commit the transactions to record a new or updated extension.
     let mut tx = conn.begin().await?;
-    let query = format!("SELECT * FROM extensions",);
-    let rows = sqlx::query(&query).fetch_all(&mut tx).await?;
+    let rows = sqlx::query!("SELECT * FROM extensions").fetch_all(&mut tx).await?;
     for row in rows.iter() {
-        extensions.push(row.get(1));
+        extensions.push(row.name.as_ref().unwrap().to_owned());
```

Comment:
I think this is going to change very soon right? This should probably return json formatted instead of just the name as a string.

Reply:
Yes, in the `cor-526` branch